views/templates.pug

```diff
@@ -49,8 +49,8 @@
     .top-bar.tn-toolbar(role="toolbar")
       table
         tr
+          td(style="padding-right: 1em")
```

Comment:
Is this a separate change? If so maybe make it on `master` separate to this? I like it :)